source-hackernews/config.yaml

```diff
@@ -0,0 +1,20 @@
+credentials:
```

Comment:
I don't think this config file is for this capture, since it doesn't look like this capture requires any configuration at all.

Reply:
is the `sops` part required?

Reply:
If the configuration has fields that need to be encrypted, yeah. There's some instructions for how to do the encryption [here](https://github.com/estuary/connectors/tree/main/python#encrypting-test-credentials). But if this capture doesn't require any kind of configuration, you could just delete the file.